### PR TITLE
V3, with peerInfo topic handling

### PR DESCRIPTION
**Pull request by 4c656554**
Added in topic handling to peer info so that hyperswarm on('connection'.. event provide a list of topics to client.

### CODE REVIEW

**Review by andrewosh — APPROVED** [review]
LGTM

File: lib/peer-info.js [hunk 1]
```diff
@@ -78,4 +82,11 @@ module.exports = class PeerInfo {
   ban (val) {
     this.banned = !!val
   }
+  
+  topic (topic) {
```
Comment (mafintosh) [inline lib/peer-info.js:86]: should be private no?

Comment (4c656554) [reply lib/peer-info.js:86]: I think it was like this in previous version, and I thought should not be private as we may need to access externally, e.g. in `.on('connection'..` (I may not properly understand public/private conventions used however).

Comment (4c656554) [reply lib/peer-info.js:86]: I see it's amended in later commit, can ignore my last comment

**Pull request merged**